corehq/apps/userreports/reports/builder/__init__.py

```diff
@@ -142,6 +147,14 @@ def make_owner_name_indicator(column_id):
     }
 
 
+def make_user_name_indicator(column_id):
+    return _make_user_group_or_location_indicator("user_id", column_id)
+
+
+def make_owner_name_indicator(column_id):
+    return _make_user_group_or_location_indicator("owner_id", column_id)
```

Comment:
You're right they aren't B3 related. I made some report builder bug fixes from the B3 QA process that I just added straight to this PR to avoid additional staging madness.